app/assets/javascripts/setup.js

```diff
@@ -17,4 +17,32 @@ $(function() {
       $(this).text('Show');
     }
   });
-});
\ No newline at end of file
+
+  $(document).on('click', '.js-toggle-registry-settings-btn', function() {
+    var targetId = $(this).data('target');
+    var collapsed = $(targetId).attr('aria-expanded') === 'true';
+
+    if (collapsed) {
+      $(this).text('Hide');
+    } else {
+      $(this).text('Show');
+    }
+  });
+
+  $(document).on('click', '.btn-add-registry', function() {
+    var registryForm = $('.registry-form'),
+        currentEntry = $(registryForm).children('.registry-entry:first'),
```

Comment:
You can simply `$registryForm.find`

Reply:
this didn't work, i also tried `$registryForm.find()` but it doesnt work anymore then

Reply:
You can leave the `children`. Just saw that it's more performatic than `find` since it only goes one level down.